purchase_report_custom_hls/models/purchase_order.py

```diff
@@ -0,0 +1,16 @@
+from odoo import api, fields, models
+
+
+class PurchaseOrder(models.Model):
+    _inherit = "purchase.order"
+
+    eta_date = fields.Char("Estimated Time of Arrival(ETA)")
+    etd_date = fields.Char("Estimated Time of Departure(ETD)")
+    quantity_total = fields.Float("Total Quantity", compute="_compute_quantity_total")
+
+    @api.onchange("order_line")
```

Comment:
```suggestion
```
Why onchange?

Reply:
Thank you.
I debug and now understand difference between onchange and depends.
Now changed to depends.